aggregate-use.rst

```diff
@@ -302,20 +302,20 @@ Preferences
 
 In the :guilabel:`Preferences` sub-tab under :guilabel:`Site Admin` tab, you can manage:
 
-- Google API credentials: These credentials are used when publishing into Google services. For details on this, see `Aggregate OAuth2 info <https://opendatakit.org/use/aggregate/oauth2-service-account/>`_
+- **Google API credentials**: These credentials are used when publishing into Google services. For details on this, see `Aggregate OAuth2 info <https://opendatakit.org/use/aggregate/oauth2-service-account/>`_
 
 .. image:: /img/aggregate-use/google-api-option.*
    :alt: Image showing google api option.
 
-- Enketo credentials: These credentials are used for Enketo webforms integration. To link Enketo with Aggregate, see `this <https://accounts.enketo.org/support/aggregate/>`_
-- ODK 2.0 App name
+- **Enketo credentials**: These credentials are used for Enketo webforms integration. To link Enketo with Aggregate, see `this <https://accounts.enketo.org/support/aggregate/>`_.
+- **ODK 2.0 App name**
       
-   - ODK Tables Synchronization Functionality - check this to enable ODK Tables functionality i.e.  able to download, upload and alter data records within ODK Tables as restricted by table-access privileges granted to the user.
+   - `ODK Tables Synchronization Functionality` - check this to enable ODK Tables functionality i.e.  able to download, upload and alter data records within ODK Tables as restricted by table-access privileges granted to the user.
 
-- Aggregate features: These settings affect the operations of the server.
+- **Aggregate features**: These settings affect the operations of the server.
    
-   - Disable faster background actions - check this to reduce AppEngine quota usage.
-   - Skip malformed submissions - check this to ignore corrupted submissions.
+   - `Disable faster background actions` - check this to reduce AppEngine quota usage.
+   - `Skip malformed submissions` - check this to ignore corrupted submissions.
```

Comment:
I'm not sure what the purpose of single backticks are...

Reply:
@adammichaelwood I added them just to make the checkbox options look different from the explanation. Should I remove them?

Reply:
Single back-ticks have no semantic meaning in RST or Sphinx. (Unlike MArkdown, where they wrap literal inline blocks.)

 - If the test is verbatim UI text from the app, use the `:guilabel:` role.
 - If they should be interpreted as `literal inline text`, use double-backticks
 - If they need to be semantically set-off, but no particular markup is appropriate, you can use *single asterisks*.